.github/workflows/e2e-tests-atomic.yml

```diff
@@ -0,0 +1,52 @@
+name: "E2E Tests on Atomic - All"
+
+# This action will run when it is triggered manually or on Pull Request with base=trunk
+on:
+  pull_request:
+    branches:
+      - trunk
+  workflow_dispatch:
+
+env:
+  WC_E2E_SCREENSHOTS: 1
+  E2E_SLACK_CHANNEL: ${{ secrets.E2E_SLACK_CHANNEL }}
+  E2E_SLACK_TOKEN: ${{ secrets.E2E_SLACK_TOKEN }}
+  E2E_WP_VERSION: 'nightly'
+  E2E_WC_VERSION: 'latest'
+  E2E_RESULT_FILEPATH: 'tests/e2e/results.json'
+  NODE_ENV: 'atomic'
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  # Run tests against WP Nightly & WC latest. Only one combination at a time.
+  wp-nightly-tests:
+    name: "WP - nightly | WC - latest | ${{ matrix.test_groups }} - ${{ matrix.test_branches }}"
+    runs-on: ubuntu-22.04
+    strategy:
+      fail-fast: false
+      max-parallel: 1
```

Comment:
Since we use the same live site for each combination, we cannot run them in parallel due to conflicts on settings etc...

Reply:
This could be an issue considering the linear nature of this, the tests would take upwards of an hour if we enable them all and run them linearly so I guess we'll always have to run a subset of tests on atomic anyways. Unless we write specific tests only for atomic.

Reply:
Yeah, that's something we'll need to think about.

Is it an issue if we just need to run them once a day (when most people are not working timezone-wise) and for the release (upon code freeze, which is on a Sunday, so it should not block anything)?

We can also focus on a set of specific smoke tests for Atomic (which should take no more than 10 or 15 mins) for faster feedback loop on some changes.